dist/constants.json

```diff
@@ -7825,697 +7825,697 @@
       },
       {
         "value": "630A4",
-        "label": "VA NEW YORK HARBOR HEALTHCARE SYSTEM - BROOKLYN DIVISION"
+        "label": "Brooklyn VA Medical Center"
       },
       {
         "value": "630A5",
-        "label": "ST ALBANS EXTENDED CARE CNTR"
+        "label": "St. Albans VA Medical Center"
       },
       {
         "value": "630GA",
-        "label": "HARLEM CBOC-NYHHS"
+        "label": "Harlem VA Clinic"
       },
       {
         "value": "630GB",
-        "label": "STATEN ISLAND CBOC-NYHHCS"
+        "label": "Staten Island Community VA Clinic"
       },
       {
         "value": "630GC",
         "label": "CHAPEL STREET OPC"
       },
       {
         "value": "632GA",
-        "label": "EAST MEADOW CBOC"
+        "label": "East Meadow VA Clinic"
       },
       {
         "value": "632HA",
-        "label": "VALLEY STREAM CBOC"
+        "label": "Valley Stream VA Clinic"
       },
       {
         "value": "632HB",
-        "label": "RIVERHEAD,NY ORC"
+        "label": "Riverhead VA Clinic"
       },
       {
         "value": "632HC",
-        "label": "BAY SHORE CBOC"
+        "label": "Bay Shore VA Clinic"
       },
       {
         "value": "632HD",
-        "label": "PATCHOGUE,NY ORC"
+        "label": "Patchogue VA Clinic"
       },
       {
         "value": "620A4",
-        "label": "VA HUDSON VALLEY HEALTH CARE - CASTLE POINT DIVISION"
+        "label": "Castle Point VA Medical Center"
       },
       {
         "value": "620GA",
-        "label": "NEW CITY"
+        "label": "New City VA Clinic"
       },
       {
         "value": "620GB",
-        "label": "PUTNAM COUNTY"
+        "label": "Carmel VA Clinic"
       },
       {
         "value": "620GD",
-        "label": "GOSHEN CBOC"
+        "label": "Goshen VA Clinic"
       },
       {
         "value": "620GE",
-        "label": "PORT JERVIS"
+        "label": "Port Jervis VA Clinic"
       },
       {
         "value": "620GF",
-        "label": "HARRIS"
+        "label": "Monticello VA Clinic"
       },
       {
         "value": "620GG",
-        "label": "POUGHKEEPSIE"
+        "label": "Poughkeepsie VA Clinic"
       },
       {
         "value": "620GH",
-        "label": "EASTERN DUTCHESS CBOC"
+        "label": "Eastern Dutchess VA Clinic"
       },
       {
         "value": "620",
-        "label": "VA HUDSON VALLEY HEALTH CARE SYSTEM - MONTROSE DIVISION"
+        "label": "Franklin Delano Roosevelt Hospital"
       },
       {
         "value": "630",
-        "label": "VA NEW YORK HARBOR HEALTHCARE SYSTEM - NEW YORK DIVISION"
+        "label": "Manhattan VA Medical Center"
       },
       {
         "value": "632",
-        "label": "NORTHPORT VAMC NY"
+        "label": "Northport VA Medical Center"
       },
       {
         "value": "526",
-        "label": "BRONX VA HOSPITAL"
+        "label": "James J. Peters Department of Veterans Affairs Medical Center"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.